Sources/Sentry/include/SentryTransaction+Private.h

```diff
@@ -5,6 +5,24 @@ NS_ASSUME_NONNULL_BEGIN
 @interface
 SentryTransaction (Private)
 
+/**
+ * The absolute system timestamp at the time this transaction began.
```

Comment:
I think we should also use absolute timestamps for transactions. I opened an issue for this https://github.com/getsentry/sentry-cocoa/issues/2155.

Reply:
👍🏻 Agreed, I also left a link there to an earlier discussion item for a refactor proposal